crates/handlers/src/graphql/mutations/user.rs

```diff
@@ -542,7 +567,10 @@ impl UserMutations {
         let user_id = NodeType::User.extract_ulid(&input.user_id)?;
```

Comment:
Isn't that fundamentally a problem with this mutation? When you have a recovery ticket, you don't know the user_id; in a sense, the recovery ticket has the user ID linked to it?

This means that either we need to make the user_id optional, or we need to make a separate mutation which only takes a recovery ticket

Reply:
Hmm. We could include the user ID in the link that's in the e-mail? But I guess that's redundant, so making the user ID optional for the recovery ticket might make sense then. I am not sure if this will turn out better or worse than a separate mutation, I don't want to duplicate the checks at least.